mods/ra/rules/player.yaml

```diff
@@ -65,4 +65,7 @@ Player:
 		Prerequisites: techlevel.infonly, techlevel.low, techlevel.medium, techlevel.unrestricted
 	GlobalUpgradeManager:
 	EnemyWatcher:
+	ProductionIconOverlay:
+		Offset: -7, 6
```

Comment:
You can tweak this a bit for additional polishing:

![image](https://cloud.githubusercontent.com/assets/756669/8392337/91c6c1a0-1ce7-11e5-945a-d434426d6869.png)